File: level80/homework/codwars8.js
//https://www.codewars.com/kata/520b9d2ad5c005041100000f/train/javascript
const pigIt = (str) => {
  console.log(str);
  return str.split(" ").map((item) => {
    let sign = "";
    item =  [...item].filter(it => {
        if(!it.match(/[,.!?;:()]/)) {
          return it;
        }
        sign = it;
      });
     let [first,...rest] = item;
     if(sign !== "" && item.length <= 1) {
       return sign;
     }
      return [...rest,first,"ay"].join("");
  }).join(" ");
}

File: level 080/homework/codwars8.py
#https://www.codewars.com/kata/520b9d2ad5c005041100000f/train/python
def pig_it(text):
    res = []
    
    for i in text.split():
        if i.isalpha():
            res.append(i[1:]+i[0]+'ay')
        else:
            res.append(i)
            
    return ' '.join(res)